daml-assistant/daml-helper/src/DA/Daml/Helper/Run.hs

```diff
@@ -319,9 +319,9 @@ runJar jarPath mbLogbackPath remainingArgs = do
 
 runDaml2ts :: [String] -> IO ()
 runDaml2ts remainingArgs = do
-  daml2ts <- fmap (</> "daml2ts" </> "daml2ts") getSdkPath
-  withProcessWait_' (proc daml2ts remainingArgs) (const $ pure ()) `catchIO`
-    (\e -> hPutStrLn stderr "Failed to invoke daml2ts." *> throwIO e)
+    daml2ts <- fmap (</> "daml2ts" </> "daml2ts") getSdkPath
```

Comment:
Is there a reason behind doing that?

Reply:
Yes. Although I ended up not adding/modifying code here in a material way, the indentation level was found to be inconsistent with the surrounding code so I fixed it.

Reply:
I'm not a huge fan of such corrections, particularly not in unrelated PRs which are already quite big.

Reply:
Ok